src/ontology/mondo-edit.obo

```diff
@@ -210861,7 +210861,7 @@ relationship: excluded_subClassOf MONDO:0019952 {source="DC-OMIM:255300", source
 [Term]
 id: MONDO:0009711
 name: congenital fiber-type disproportion myopathy
-def: "A rare genetic disorder caused by mutations in the TPM3, ACTA1, RYR1 or SEPN1 genes. It is inherited in an autosomal dominant or recessive pattern and rarely in an X-linked pattern. It manifests with myopathy throughout the body, particularly in the muscles of the shoulders, upper arms, hips, and thighs. Affected individuals may have contractures, lordosis, or scoliosis. In a minority of cases mild to severe breathing problems may occur." [NCIT:C120046]
+def: "A rare genetic disorder caused by mutations in the TPM3, ACTA1, RYR1 and SEPN1 genes. It is inherited in an autosomal dominant or recessive pattern and rarely in an X-linked pattern. It manifests with myopathy throughout the body, particularly in the muscles of the shoulders, upper arms, hips, and thighs. Affected individuals may have contractures, lordosis, or scoliosis. In a minority of cases mild to severe breathing problems may occur." [NCIT:C120046]
```

Comment:
@katiermullen does this require a mutation in ALL of these genes? or in one of them or in one or more of them?

Reply:
@sabrinatoro I just came across this discrepancy between the mondo definition and the NCI definition where the Mondo definition came from when I was researching this issue. Although the source definition uses "and" upon further research, I think "or" is correct based on this reference PMID:31578728, which I can include as a db-xref in the definition. 

One question though - are the SC axioms ok since they are not always true?
<img width="213" alt="Screenshot 2024-12-20 at 1 58 18 PM" src="https://github.com/user-attachments/assets/4bf245ca-49d4-4549-9577-f301f17c00e9" />

Thank you for your help!